### ISSUE TITLE
SmartCity-Accessibility #219 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#147]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 53 lines...]


Downloading: https://jitpack.io/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml
Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml


[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ SmartCity-Accessibility ---
[INFO] Deleting /var/lib/jenkins/workspace/SmartCity-Accessibility/target
[INFO] 
[INFO] --- jacoco-maven-plugin:0.7.8:prepare-agent (default-cli) @ SmartCity-Accessibility ---
[INFO] argLine set to -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/workspace/SmartCity-Accessibility/target/jacoco.exec
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ SmartCity-Accessibility ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ SmartCity-Accessibility ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 63 source files to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/classes
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/NearbyPlacesAttempt.java: Some input files use or override a deprecated API.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/NearbyPlacesAttempt.java: Recompile with -Xlint:deprecation for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/gui/components/ButtonsPanel.java:[13,45] cannot find symbol
  symbol:   class jxMapsFunctionalityTest
  location: package smartcity.accessibility.mapmanagement
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 15.076 s
[INFO] Finished at: 2017-01-09T17:41:14+00:00
[INFO] Final Memory: 29M/143M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/gui/components/ButtonsPanel.java:[13,45] cannot find symbol
[ERROR] symbol:   class jxMapsFunctionalityTest
[ERROR] location: package smartcity.accessibility.mapmanagement
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Ariel Kolikant] 083704e5cf277ff1b02dfe9777f2cc0d5a3a34b5 - #129 now marks adress searches:

- [Ariel Kolikant] 1b7142f40c4237b8fc850e92be8f48a6a630739e - all markers created by jxmapsfunctionality can now be cleared by button

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/219/)

**Comment by Kolikant** [comment]
@KaplanAlexander 
translation?

**Comment by kapalex** [comment]
`[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/gui/components/ButtonsPanel.java:[13,45] cannot find symbol
  symbol:   class jxMapsFunctionalityTest
  location: package smartcity.accessibility.mapmanagement`
 
look at that line and see the issue
my guess is you forgot to add some file in the commit

**Comment by Kolikant** [comment]
is it not finding jxmaps....test or not finding button panel?
both of which have long been commited

**Comment by kapalex** [comment]
buttonspanel.java line 13
seems like your importing a test file for some reason?
probably that causes a problem with maven

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 53 lines...]


Downloading: https://jitpack.io/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml
Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml


[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ SmartCity-Accessibility ---
[INFO] Deleting /var/lib/jenkins/workspace/SmartCity-Accessibility/target
[INFO] 
[INFO] --- jacoco-maven-plugin:0.7.8:prepare-agent (default-cli) @ SmartCity-Accessibility ---
[INFO] argLine set to -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/workspace/SmartCity-Accessibility/target/jacoco.exec
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ SmartCity-Accessibility ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ SmartCity-Accessibility ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 63 source files to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/classes
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/NearbyPlacesAttempt.java: Some input files use or override a deprecated API.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/NearbyPlacesAttempt.java: Recompile with -Xlint:deprecation for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/gui/components/ButtonsPanel.java:[13,45] cannot find symbol
  symbol:   class jxMapsFunctionalityTest
  location: package smartcity.accessibility.mapmanagement
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 15.419 s
[INFO] Finished at: 2017-01-09T17:49:09+00:00
[INFO] Final Memory: 26M/165M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/gui/components/ButtonsPanel.java:[13,45] cannot find symbol
[ERROR] symbol:   class jxMapsFunctionalityTest
[ERROR] location: package smartcity.accessibility.mapmanagement
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Ariel Kolikant] 083704e5cf277ff1b02dfe9777f2cc0d5a3a34b5 - #129 now marks adress searches:

- [Ariel Kolikant] 1b7142f40c4237b8fc850e92be8f48a6a630739e - all markers created by jxmapsfunctionality can now be cleared by button

- [Ariel Kolikant] 779568aa281d3afa56b5c48ad5d5c2ef26485ebf - is this it jenkins?



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/220/)

**Comment by Kolikant** [comment]
oh, okay. thank you. I think I know now what it is then

**Comment by JuanJenkinsBot** [comment]
Build was fixed!

**Comment by Kolikant** [comment]
hush now Jenkins. you are safe.
it can't hurt you anymore
